tox.ini

```diff
@@ -19,10 +20,17 @@ deps =
     dj32: Django==3.2.*
     dj40: Django==4.0.*
     dj41: Django==4.1.*
+    dj41: Django==4.2.*
     djmain: https://github.com/django/django/archive/main.tar.gz
     -rtest_project/requirements.txt
 
-passenv = DISPLAY XAUTHORITY XDG_* PIP_* BROWSER MOZ_HEADLESS
+passenv =
+    DISPLAY
+    XAUTHORITY
+    XDG_*
+    PIP_*
+    BROWSER
+    MOZ_HEADLESS
```

Comment:
required for tox 4 compatibility: https://tox.wiki/en/latest/upgrading.html#changed-ini-rules